Library/Formula/libgdiplus.rb

```diff
@@ -0,0 +1,32 @@
+class Libgdiplus < Formula
+  desc "C-based implementation of the GDI+ API for Mono"
+  homepage "http://www.mono-project.com/docs/gui/libgdiplus/"
+  url "https://github.com/mono/libgdiplus/tarball/3b284ab28cb8737f9d14dabfedc6903655c66a7f"
```

Comment:
Thanks.

@directhex / Jo, Could you please tag the currently merged state in the git repository? Thank you. :)